app/views/user_mailer/shared/_in_person_ready_to_verify.html.erb

```diff
@@ -80,16 +80,16 @@
     <table class="margin-bottom-3">
       <tbody>
         <tr>
-          <td align="left" width="110.46px">
+          <td align="left" style="width:110px;">
```

Comment:
I'm sorta curious if it's necessary to specify the width at all, or would the cell grow automatically to accommodate the image's defined dimensions? The only things that gives me pause here is:

1. Usually `style` attributes won't work in production due to [CSP `style-src`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src), but that's _probably_ not an issue for emails, particularly given the second point below.
2. Rails mailers will automagically inline CSS styling as `style` attribute, but I'm not sure how it works if we also specify our own `style` attribute value.

(This isn't a blocker)

Reply:
@aduth  Thanks for the comments here. I need to dig a little more but here is what I know so far...


The cell does not grow to accommodate the images defined dimensions. When I removed `style="width:110px;` on line 54- you can see the Real ID image below Option 1 shrunk. 

<img width="708" alt="Screenshot 2024-06-20 at 2 00 30 PM" src="https://github.com/18F/identity-idp/assets/125507397/20143496-7206-489f-ba18-1edf2b7556ce">

![Screenshot 2024-06-20 at 1 53 24 PM](https://github.com/18F/identity-idp/assets/125507397/96ca9162-4fd7-4a40-ae32-ef82c6ef30d0)

Worth mentioning- I created an enrollment in staging and the email I received did not have working images on it. (This EIPP email is only in staging.) I was thinking it was the decimal pixels but am wondering (and will look into) CSP.

Reply:
Ohhhh, I know what it is. You're trying to use SVG images but those aren't broadly supported:

https://www.caniemail.com/features/image-svg/

This is also why we have `app/assets/images/email` folder, which includes copies of images specifically because of this support limitation (noted also in folder README):

https://github.com/18F/identity-idp/tree/main/app/assets/images/email

Reply:
Great tip! Okay, each image on the email template is pointing at a png file inside app/assets/images/email. I did not delete the svg files because they each are being used in app/views/idv/in_person/ready_to_verify/show.html.erb.